bears/natural_language/WriteGoodLintBear.py

```diff
@@ -71,7 +71,7 @@ def create_arguments(filename, file, config_file,
             'allow_extra_words': '--tooWordy',
             'allow_cliche_phrases': '--cliches'
         }
-        l = locals()
+        L = locals()
```

Comment:
this bypasses the error, but doesnt fix the problem.  The variable needs a meaningful name.  `arg_values` is my suggestion.